pkg/graph/schema/schema.graphql

```diff
@@ -1,1243 +1,14 @@
-  status: OperationalSolutionSubtaskStatus!
-}
-
-type OperationalSolutionSubtask {
-  id: UUID!
-  solutionID: UUID!
-  name: String!
-  status: OperationalSolutionSubtaskStatus!
-
-  createdBy: UUID!
-  createdByUserAccount: UserAccount!
-  createdDts: Time!
-  modifiedBy: UUID
-  modifiedByUserAccount: UserAccount
-  modifiedDts: Time
-}
-
-type DiscussionRoleSelection {
-  userRole: DiscussionUserRole!
-  userRoleDescription: String
-}
-"""
-The inputs to the user feedback form
-"""
-input SendFeedbackEmailInput {
-  isAnonymousSubmission: Boolean!
-  allowContact: Boolean
-  cmsRole: String
-  mintUsedFor: [MintUses!]
-  mintUsedForOther: String
-  systemEasyToUse: EaseOfUse
-  systemEasyToUseOther: String
-  howSatisfied: SatisfactionLevel
-  howCanWeImprove: String
-}
-
-enum EaseOfUse {
-  AGREE
-  DISAGREE
-  UNSURE
-}
-
-enum MintUses {
-  VIEW_MODEL
-  EDIT_MODEL
-  SHARE_MODEL
-  TRACK_SOLUTIONS
-  CONTRIBUTE_DISCUSSIONS
-  VIEW_HELP
-  OTHER
-}
-enum SatisfactionLevel {
-  VERY_SATISFIED
-  SATISFIED
-  NEUTRAL
-  DISSATISFIED
-  VERY_DISSATISFIED
-}
-
-
 """
 Query definition for the schema
 """
 type Query {
-  planDocument(id: UUID!): PlanDocument!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  existingModelCollection: [ExistingModel!]!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
   searchOktaUsers(searchTerm: String!): [UserInfo!]!
   @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  planCollaboratorByID(id: UUID!): PlanCollaborator!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  taskListSectionLocks(modelPlanID: UUID!): [TaskListSectionLockStatus!]!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  planPayments(id: UUID!): PlanPayments!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  ndaInfo: NDAInfo!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  planCR(id: UUID!): PlanCR!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  planTDL(id: UUID!): PlanTDL!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  operationalSolutions(operationalNeedID: UUID!, includeNotNeeded: Boolean! = false): [OperationalSolution!]!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  operationalSolution(id: UUID!): OperationalSolution!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  operationalNeed(id: UUID!): OperationalNeed!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  auditChanges(tableName: String!, primaryKey: UUID!): [AuditChange!]!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  possibleOperationalNeeds: [PossibleOperationalNeed!]!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  possibleOperationalSolutions: [PossibleOperationalSolution!]!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  userAccount(username: String!): UserAccount!
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
-  existingModelLink(id: UUID!): ExistingModelLink!
-  @hasAnyRole(roles:[MINT_USER, MINT_MAC])
-  mostRecentDiscussionRoleSelection: DiscussionRoleSelection
-  @hasAnyRole(roles: [MINT_USER, MINT_MAC])
 }
 
 """
 Mutations definition for the schema
 """
 type Mutation {

```

Comment:
I tested this, the dummy mutation isn't necesary. This works, I tested it out locally. See this [GitHubIssue](https://github.com/99designs/gqlgen/issues/630#issuecomment-473735885)
```suggestion
type Mutation
```

This way, we don't have a mutation that panics as expected behavior